.github/workflows/ci-kubeadm.yml

```diff
@@ -3,8 +3,14 @@ name: ci-test
 on:
   push:
     branches: [main]
+    paths: 
+    - 'KubeArmor/**'
+    - 'tests/**'
   pull_request:
     branches: [main]
+    paths: 
```

Comment:
I guess we might have to consider deployments folder too? Since it has yamls which if changed needs to be validated by tests.

Reply:
These tests are based on KubeArmor as a process not deployment, so I don't think they are relevant in this case.

Reply:
@Ankurk99  is adding tests based on images ... will this not impact?

Reply:
They will be in a separate action(file)